x-pack/plugins/cases/public/components/edit_connector/index.tsx

```diff
@@ -260,27 +204,35 @@ export const EditConnector = React.memo(
 
     const connectorIdConfig = getConnectorsFormValidators({
       config: schema.connectorId as FieldConfig,
-      connectors,
+      connectors: allAvailableConnectors,
     });
 
-    const { pushButton, pushCallouts } = usePushToService({
-      connector: {
-        ...caseData.connector,
-        name: isEmpty(connectorName) ? caseData.connector.name : connectorName,
-      },
-      caseServices,
+    const connectorWithName = {
+      ...caseData.connector,
+      name: isEmpty(actionConnector?.name) ? caseData.connector.name : actionConnector?.name ?? '',
+    };
+
+    const {
+      errorsMsg,
+      needsToBePushed,
+      hasBeenPushed,
+      isLoading: isLoadingPushToService,
+      hasPushPermissions,
+      hasErrorMessages,
+      hasLicenseError,
+      handlePushToService,
+    } = usePushToService({
```

Comment:
`usePushToService` returns all the logic needed by the component and it does not return any components.